content/en/docs/ops/ambient/usage/ztunnel/index.md

```diff
@@ -0,0 +1,656 @@
+It is also worth noting that in Ambient mode, a simplified set of resources are used in the xDS apis for ztunnel proxy configuration. This results in improved performance (having to transmit and process a much smaller set of information that is sent from `istiod` to the ztunnel proxies) and improved troulbeshooting. 
+
+### Data plane overview
+
+Having briefly described the control plane architecture, we now briefly summarize key aspects of the data plane functional behavior. 
+
+#### Ztunnel to ztunnel datapath
+The first scenario is direct ztunnel to ztunnel L4-only networking. This is depicted in the following figure.
+  
+{{< image width="100%"
+    link="ztunnel-datapath-1.png"
+    caption="Basic ztunnel L4-only datapath"
+    >}}
+
+The figure depicts ambient pod workloads running on two nodes W1 and W2 of a Kubernetes cluster. There is a single instance of the ztunnel proxy on each node. In this scenario, application client pods C1, C2 and C3 need to access a service provided by pod S1 and there is no requirement for advanced L7 services such as L7 traffic routing or L7 traffic manaagementand hence no Waypoint proxy needed in the datapath.    
+
+The figure shows that pods C1 and C2 running on node W1 connect with pod S1 running on node W2 and their TCP traffic is tunneled through a single shared HBONE tunnel instance that has been created between the ztunnel proxy pods of each node. Mutual TLS (mTLS) is used for encryption as well as mutual authentication of traffic getting tunneled. SPIFFE identities are used as the workload identities for each side as in sidecar based Istio.  The term `HBONE` (for HTTP Based Overlay Network Encapsulation) is used in Istio Ambient to refer to a technique for transparently and securely tunneling TCP packets encapsulated within HTTPS packets. Some brief additional notes on HBONE are provided in a following subsection. 
+
+We note that the figure also shows traffic from pod C3 destined to destination pod S1 on worker node W2 also traverses the local ztunnel proxy instance so that L4 traffic management functions such as L4 Authorization and L4 Telemetry are enforced identically on traffic whether or not it crosses a node boundary. 
+
+#### Ztunnel datapath via waypoint
+The next figure illustrates the data path for a different use case where there is a need to traverse through an interim Waypoint proxy. This is the case when the application service requires advanced L7 traffic routing, management or policy handling. Here ztunnel L4 networking is used to tunnel traffic torwads a Waypoint proxy for L7 processing and then mapped via a second HBONE tunnel towards the ztunnel on the node hosting the selected service destination pod.  In general the Waypoint proxy may or may not be located on the same nodes as the source or destinaton pods. 
+
+{{< image width="100%"
+    link="ztunnel-waypoint-datapath.png"
+    caption="Ztunnel datapath via an interim waypoint"
+    >}}
+
+
+#### Ztunnel datapath hairpinning 
```

Comment:
It may be better to leave this section out. It seems like the details and plans of hair pinning are too fluid for this type of document. It's an interesting area with lots of discussion but at least to my mind it's just not really pinned down enough yet.

Reply:
This is the current behavior. Since this guide is meant to help alpha testers who will encounter this functionality in Istio 1.18, 1.19 and possibly permanently, I think it might be better to leave this in the guide until such a time that the community does decide to change it. To be clear, I am adding an additional statement that this behavior is currently under review and could change in future.

Reply:
Added a warning blurb as follows .. to make this more explicit for hair-pinning even though there is already a disclaimer at the top of the guide about any of the functionality potentially changing as the project evolves to beta and beyond.
_{{< warning >}}
As noted earlier, some ambient functions may change as the project moves to betta status and beyo
nd. This feature (hair-pinning) is an example of a feature that is currently available in the alpha version of ambient and under review for modification as the project evolves.
{{< /warning >}}_